plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```diff
@@ -759,11 +759,31 @@ export class SessionRecordingIngester {
                 const lastKnownCommit = metrics.lastKnownCommit ?? -1
 
                 if (!highestOffsetToCommit) {
+                    status.warn('🤔', 'blob_ingester_consumer - no highestOffsetToCommit for partition', {
```

Comment:
_today_ we can log this only when the partition is 101 if we want to avoid too much noise

Reply:
ah good idea